src/pages/Step.tsx

```diff
@@ -36,9 +34,7 @@ const NumberIndicator = ({ children, ...props }: NumberIndicatorProps) => {
   );
 };
 
-const Step = ({ stepNumber, children, ...props }: StepProps) => {
-  const { t } = useTranslation();
-
+const Step = ({ stepNumber, children, title = '', ...props }: StepProps) => {
```

Comment:
```suggestion
const Step = ({ stepNumber, children, title, ...props }: StepProps) => {
```

Instead of initializing `title = ''` you can use defaultProps like everywhere else